.github/workflows/bandit.yml

```diff
@@ -0,0 +1,16 @@
+name: Linter
+on: pull_request
+jobs:
+  Bandit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Run checks
+        run: |
+          pip install --user -r <(grep "^bandit" ./requirements.txt)
+          echo "Bandit version: "`bandit --version | head -1`
```

Comment:
I won't say it is a problem, but this line looks a little bit extra, because the previous line outputs the version installed.